COMPATIBILITY.md

```diff
@@ -0,0 +1,16 @@
+The following table shows the compatibility of jaeger operator with different components, in this particular case we shows Kubernetes and Strimzi operator compatibility
+
+
+| Jaeger Operator | Kubernetes           | Strimzi Operator   |
```

Comment:
Should we then replace this table with a link?
https://github.com/jaegertracing/jaeger-operator#compatibility-matrix

Reply:
We can do this in a separate PR, I would prefer to use this PR only for the release.

Reply:
Should we then update the Readme too? Thats the first thing people see when they go on the github page.